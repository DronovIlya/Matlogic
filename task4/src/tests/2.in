?x!P(x)|-!@xP(x)
(P(x)&G(x))->(P(x)&G(x))->(P(x)&G(x))