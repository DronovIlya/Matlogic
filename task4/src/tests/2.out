((((P(x))&(G(x))))->(((((P(x))&(G(x))))->(((P(x))&(G(x)))))))
((((((P(x))&(G(x))))->(((((P(x))&(G(x))))->(((P(x))&(G(x))))))))->(((?x((!(P(x)))))->(((((P(x))&(G(x))))->(((((P(x))&(G(x))))->(((P(x))&(G(x)))))))))))
((?x((!(P(x)))))->(((((P(x))&(G(x))))->(((((P(x))&(G(x))))->(((P(x))&(G(x)))))))))
